packages/sanity/src/core/releases/tool/__tests__/BundlesOverview.test.tsx

```diff
@@ -1,9 +1,11 @@
 import {beforeEach, describe, expect, it, jest} from '@jest/globals'
```

Comment:
off the bat and already writing tests we do love to see it WE DO LOVE TO SEE IT.

Reply:
Hats of for @jordanl17

Reply:
@jordanl17 👏 👏